lib/domain/contact_manager/contacts_manager.dart

```diff
@@ -88,79 +70,29 @@ class ContactsManager {
     _getAllContactsInteractor
         .execute(limit: AppConfig.maxFetchContacts)
         .listen((event) {
-      event.fold(
-        (_) {},
-        (success) {
-          if (success is GetContactsSuccess) {
-            _tomContacts = success.tomContacts.toSet();
-            _phonebookContacts = {};
-            _syncContactsState = SyncContactsState.synchronized;
-            Logs().d(
-              "ContactManagerMixin()::getAllContacts(): TomContacts: ${_tomContacts.length}",
-            );
-            Logs().d(
-              "ContactManagerMixin()::getAllContacts(): PhonebookContacts: ${_phonebookContacts.length}",
-            );
-          }
-        },
-      );
-
-      contactsStream.add(event);
+      contactsNotifier.value = event;
+    }).onDone(_fetchPhonebookContacts);
+  }
+
+  void _fetchPhonebookContacts() async {
+    if (!PlatformInfos.isMobile ||
+        await _permissionHandlerService.contactsPermissionStatus !=
+            PermissionStatus.granted) {
+      return;
```

Comment:
how to make UI update it well in this case?

Reply:
do nothing for this case, also have no design for it